cdk/stacks/wdiv_s3_trigger_stack.py

```diff
@@ -82,6 +82,9 @@ def get_lambda_env_vars(self):
             "WDIV_WEBHOOK_URL": ssm.StringParameter.value_for_string_parameter(
                 self, "/wdiv_s3_trigger/WDIV_WEBHOOK_URL"
             ),
+            "SERVER_ENVIRONMENT": ssm.StringParameter.value_for_string_parameter(
+                self, "/wdiv_s3_trigger/SERVER_ENVIRONMENT"
```

Comment:
`/wdiv_s3_trigger/SERVER_ENVIRONMENT` doesn't exist in SSM, but `self.dc_environment` does. 

This should be 
```python
"SERVER_ENVIRONMENT": self.dc_environment
```

Reply:
And the value will be one of these https://github.com/DemocracyClub/UK-Polling-Stations/blob/master/cdk/app.py#L12-L16